imi/src/main/java/org/motechproject/nms/imi/exception/NotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package org.motechproject.nms.imi.exception;
+
+public class NotFoundException extends IllegalStateException {
+    public NotFoundException(String message) {
+        super(message);
```

Comment:
Just brainstorming here. If this is being used as a wrapper to throw 404 instead of 400 for a file request, I wonder if 404 is the right response for it.

Ideally, 404 would be a response to a GET method where we say we can't find a resource that they are trying to get... but if they are POSTing to us to go find files that don't exist, it should be considered a bad request, right?